src/BaseApiAbstract.php

```diff
@@ -9,10 +9,10 @@
 use GuzzleHttp;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
 use RuntimeException;
 use Smartling\AuthApi\AuthApiInterface;
 use Smartling\Exceptions\SmartlingApiException;
-use Smartling\Logger\DevNullLogger;
```

Comment:
Removed custom null logger class and used from `psr/log` to be compatible with `psr/log 3`.